src/pudl/metadata/fields.py

```diff
@@ -1552,7 +1552,7 @@
         "description": "For nuclear plants only, the unit number .One digit numeric. Nuclear plants are the only type of plants for which data are shown explicitly at the generating unit level.",
     },
     "num_transmission_circuits": {
-        "type": "integer",
+        "type": "number",
```

Comment:
Is it possible to have a non-integer number of transmission circuits?  If not, then I think we probably want to transform the input strings such that they're compatible with integer types. We've done this elsewhere by using something like:

```py
df["my_int_col"] = pd.to_numeric(df.floaty_int_strings).astype(int)
```

`pd.to_numeric()` will happily go from string to float, and assuming all the floats can be represented as ints, the `astype()` will be successful.